Sigma_specification.md

```diff
@@ -285,27 +285,38 @@ We recommend using a "definition" value in cases in which further explanation is
 * product - examples: windows, apache, check point fw1
 * service - examples: sshd, applocker
 
-The "category" value is used to select all log files written by a certain group of products, like firewalls or web server logs.
-The automatic converter will use the keyword as a selector for multiple indices.
+The `category` value is used to select all log files written of a logical group. \
+This may cover one or more sources of information depending on the system. \
+e.g. "antivirus" for the scan result, "webserver" for the web access logs. 
 
-The "product" value is used to select all log outputs of a certain product, e.g. all Windows Eventlog types including "Security", "System", "Application" and the new log types like "AppLocker" and "Windows Defender".
+The `product` value is used to select all log outputs of a certain product. \
+It can be as generic as an operating system or the name of a particular software package. \
+e.g. "windows" will include "Security", "System", "Application" and the other like "AppLocker" and "Windows Defender"...
 
-Use the "service" value to select only a subset of a product's logs, like the "sshd" on Linux or the "Security" Eventlog on Windows systems.
+The `service` value is used to select a more specific subset of logs. \
+e.g. "sshd" on Linux or the "Security" Eventlog on Windows systems.
```

Comment:
I think this is a bit confusing because we have a different writing of the service `security` for `Security` logs in Windows. I recommend to adapt the example as well as giving the rule writer the advice that the case is usually lower case and spaces are replaced by underlines, if not defined otherwise for any reason.

Reply:
I'll go into a bit more detail